README.md

```diff
@@ -9,7 +9,7 @@ LexikJWTAuthenticationBundle
 
 This bundle provides JWT (Json Web Token) authentication for your Symfony API.
 
-It is compatible and tested with PHP 5.5, 7.2, 7.3, on Symfony 3.4 and 4.2.
+It is compatible and tested with PHP 5.6, 7.2, 7.3, on Symfony 3.4 and 4.2.
```

Comment:
Maybe add Symfony 5.1 if you are testing it?
(i plan to test it an on one of my 5.1 project when 5.1 is release, could do it then if not now :))

Reply:
Done!